src/components/NavBar/NavBar.js

```diff
@@ -1 +1,13 @@
-import React from 'react';
\ No newline at end of file
+import React from 'react';
+
+export const NavBar = () => {
+  return (
+    <div className='NavBar'>
```

Comment:
Do we want to hook up Router and change these buttons to NavLinks? I'm a little confused about the videos button, and I think we decided the calendar was not MVP. We should touch base on how these will work.

Reply:
Yes absolutely! Good call - I'll delete the calendar and videos button, and we can implement Router together this afternoon!